daprdocs/content/en/reference/cli/dapr-mtls/_index.md

```diff
@@ -40,6 +41,7 @@ You can learn more about each sub command from the links below.
 
 - [`dapr mtls expiry`]({{< ref dapr-mtls-expiry.md >}})
 - [`dapr mtls export`]({{< ref dapr-mtls-export.md >}})
+- [`dapr mtls renew-certificate`]({{< ref dapr-mtls-renew-certificate.md >}})
 
 ### Examples
 
```

Comment:
Can you update the comment in the example below to say "for K8s" like this

# Check if mTLS is enabled on the Kubernetes cluster
dapr mtls -k